bot/constants.py

```diff
@@ -23,6 +23,7 @@
     "Reddit",
     "RedisConfig",
     "RedirectOutput",
+    "PYTHON_PREFIX"
```

Comment:
I just started on this project, AFAIK, `__all__` is used for wrapping classes under constants.py.

If true, we can remove `PYTHON_PREFIX` from `__all__`